frontend/package.json

```diff
@@ -68,6 +74,21 @@
       "react-hooks/exhaustive-deps": "off"
     }
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.+(js|jsx|ts|tsx)": [
+      "eslint --fix",
+      "git add"
+    ],
+    "*.+(json|css|md)": [
+      "prettier --write",
+      "git add"
+    ]
+  },
```

Comment:
This is the main automation piece:
1. Husky will run **lint-staged** on every file to be committed
1. Lint-staged will lint all javascript/typescript files (.js and .ts) that are staged prior to their commit

The above is an optimization of just using Husky and runnint ESLint on the whole code-base on each commit. That can get slow as the app grows. So with lint-staged we only check the new files. 

That is assuming your existing code-base is already linted. That's why I also submitted the lint changes for prior files with this PR.

See:
https://medium.com/@okonetchnikov/make-linting-great-again-f3890e1ad6b8
https://thomlom.dev/setup-eslint-prettier-react/